tests/test_prompt_system.py

```
@@ -0,0 +1,493 @@
+        """Prompt file should take priority over task."""
+        manager = PromptManager()
+        with tempfile.NamedTemporaryFile(mode="w", suffix=".txt", delete=False) as f:
+            f.write("Override prompt")
+            f.flush()
+            prompt = manager.get_prompt(task="audiobook", prompt_file=f.name)
+        assert prompt == "Override prompt"
+        Path(f.name).unlink()
+
+    def test_validate_prompt_valid(self):
+        manager = PromptManager()
+        is_valid, msg = manager.validate_prompt("A valid prompt with enough text")
```

Ruff (0.15.9)

[warning] Unpacked variable `msg` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)